src/pages/api/coverage.ts

```diff
@@ -4,12 +4,12 @@ import { NextApiRequest, NextApiResponse } from 'next';
 const globalAny: any = global;
 
 export default (req: NextApiRequest, res: NextApiResponse): void => {
-    if (process.env.NODE_ENV === 'production') {
+    if (process.env.NODE_ENV === 'production' && !process.env.NEXT_PUBLIC_CYPRESS) {
         res.status(404);
     }
     else {
         res.status(200).json({
             coverage: globalAny.__coverage__ || null,
         });
     }
-};
\ No newline at end of file
+};
```

Comment:
It looks like your editor has added a trailing newline in the file. This is not a big problem in itself, but it tells me that maybe your editor has not picked up the settings from the `.editorconfig` file. See the README for instructions/links on how to set that up. 😊